README.md

```diff
@@ -32,7 +32,7 @@ C++ implementation of [Apache Iceberg™](https://iceberg.apache.org/).
 
 ```bash
 cd iceberg-cpp
-cmake -S . -B build -DCMAKE_INSTALL_PREFIX=/path/to/install -DICEBERG_BUILD_STATIC=ON -DICEBERG_BUILD_SHARED=ON
+cmake -S . -B build -DCMAKE_INSTALL_PREFIX=/path/to/install -DICEBERG_BUILD_STATIC=ON -DICEBERG_BUILD_SHARED=ON -DICEBERG_BUILD_TESTS=OFF -DICEBERG_ARROW=OFF
```

Comment:
Maybe these options should be documented somewhere but not specified in the  same example command?

Reply:
Agreed. An official site for iceberg-cpp is more suitable for these documentations.

Reply:
Should we open an issue to choose a documentation/site generator?

Reply:
It would be good if someone with expertise could lead this :)

Reply:
https://github.com/apache/iceberg-cpp/issues/26

Reply:
This PR is to update cmake to build Core Libraries, and I think we don't need to build `Arrow` for the core libraries, right?